src/KafkaFlow/KafkaMessage.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace KafkaFlow;
+
+public class KafkaMessage<TPayload> 
+{
+    public KafkaSchema Schema { get; set; } = new();
+    public OutboxPayload<TPayload> Payload { get; set; } = new();
```

Comment:
@eduardosbcabral , aqui você ta assumindo que a mensagem sempre vai ter esse formato do **OutboxPayload**?

Se sim, não é bem verdade. Por exemplo, o evento que a merchant-api emite já não está nesse formato.

Reply:
Isso, faz mais sentido deixar quem for usar definir o contrato :)